public/shared/messages.js

```diff
@@ -31,10 +31,30 @@ const ERROR_DELETING_DATASET_MESSAGE =
 const SUCCESS_DELETING_DATASET_MESSAGE = 'The dataset was deleted successfully';
 const SUCCESS_DELETING_RESULT_MESSAGE =
   'The result dataset was deleted successfully';
-  const SUCCESS_DELETING_ALGORITHM_MESSAGE =
+const SUCCESS_DELETING_ALGORITHM_MESSAGE =
   'The algorithm was deleted successfully';
 const ERROR_DELETING_RESULT_MESSAGE =
   'There was an error deleting the result dataset';
+const ERROR_PYTHON_NOT_INSTALLED_MESSAGE =
+  'Python is not installed on your system';
+const buildPythonWrongVersionMessage = (version) =>
+  `Your system's python version (${version}) is not supported`;
```

Comment:
Use: _Your system is running Python version ${version}, which is not supported by this application. Please upgrade to Python 2.7.0 or better._